trident/absorption_spectrum/absorption_spectrum.py

```diff
@@ -925,7 +947,10 @@ def _add_lines_to_spectrum(self, field_data, use_peculiar_velocity,
                 # normal use of the word by observers.  It is an equivalent
                 # with in tau, not in flux, and is only used internally in
                 # this subgrid deposition as EW_tau.
-                vEW_tau = vtau * vbin_width[i]
+                if lambda_0 == 2.1e9:
+                    vEW_tau = vtau
```

Comment:
A comment here would be useful to explain why the behavior is different for this method.